aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -511,6 +523,7 @@ bool can_use_cudnn_attention(const sdp_params& params, bool debug) {
   constexpr auto general_constraints =
       array_of<bool (*)(sdp_params const&, bool)>(
           check_runtime_enabled_cudnn,
+          check_runtime_disabled_cudnn,
```

Comment:
might want to unify this with `check_runtime_enabled_cudnn` at some point haha

Reply:
Indeed, but my hope is that we will eventually be able to completely remove `check_runtime_enabled_cudnn` so I am just incentivizing us lol